HHS/OPRE-OPS
# Add CANAppropriationDetails model and CANStatus enum

## What changed

Adding CANAppropriationDetails model and table. Additionally, added the CANStatus enum with two values: Active and Inactive. This is now present on the CAN as the status property, which is calculated based on a combination of funding and expiration date. 
## Issue

[OPS-2059](https://github.com/HHS/OPRE-OPS/issues/2059)

## How to test

Start the app and look at the can_appropriation_details table in the database. There should be 3 rows.
Run unit tests

## Definition of Done Checklist
